pkg/validate/validate.go

```diff
@@ -157,20 +156,5 @@ func validateInstallPackagePath(path util.Path, val interface{}) util.Errors {
 		return util.NewErrs(fmt.Errorf("invalid value %s:%s", path, valStr))
 	}
 
-	validPrefixes := []string{util.LocalFilePrefix}
-	validPrefixDesc := map[string]string{
-		util.LocalFilePrefix: "RFC 8089 absolute path to file e.g. file:///var/istio/config.yaml",
-	}
-
-	for _, vp := range validPrefixes {
-		if strings.HasPrefix(valStr, vp) {
-			return nil
-		}
-	}
-
-	errStr := fmt.Sprintf("Invalid path prefix for %s: %s. \nMust be one of the following:\n", path, valStr)
-	for _, vp := range validPrefixes {
-		errStr += fmt.Sprintf("%-15s %s", vp, validPrefixDesc[vp])
-	}
-	return util.NewErrs(fmt.Errorf(errStr))
+	return nil
```

Comment:
Not sure why we remove the validation for local file path? Do we still support local installation path

Reply:
Yes, but we've removed the file:/// prefix.